vm-azure-csharp/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Should the entire `.vscode` directory be included in this PR? It would think this is something we want to exclude.

Reply:
Whoa! no, it should be excluded. Good catch. I'll also add to .gitignore.